### PR TITLE
Add cases: Forbid using relative path on blockcopy

### PR DESCRIPTION
**Pull request by qiankehan**
Negative tests for https://bugzilla.redhat.com/show_bug.cgi?id=1300177

### CODE REVIEW

**Comment by qiankehan** [comment]
Test result:
$ sudo avocado run --vt-type libvirt type_specific.io-github-autotest-libvirt.virsh.blockcopy.negative_test.relative_path
JOB ID     : e3dfbc4251c74b636f17c545bd2b816fb2f5d9f5
JOB LOG    : /root/avocado/job-results/job-2017-04-19T11.09-e3dfbc4/job.log
TESTS      : 3
 (1/3) type_specific.io-github-autotest-libvirt.virsh.blockcopy.negative_test.relative_path.image_name_only: PASS (10.55 s)
 (2/3) type_specific.io-github-autotest-libvirt.virsh.blockcopy.negative_test.relative_path.parent_dir: PASS (12.77 s)
 (3/3) type_specific.io-github-autotest-libvirt.virsh.blockcopy.negative_test.relative_path.current_dir: PASS (14.71 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0
JOB HTML   : /root/avocado/job-results/job-2017-04-19T11.09-e3dfbc4/html/results.html
TESTS TIME : 38.03 s

File: libvirt/tests/cfg/virsh_cmd/domain/virsh_blockcopy.cfg [hunk 1]
```diff
@@ -170,3 +170,12 @@
...
                     virsh_uri = "qemu:///system"
+                - relative_path:
+                    bug_url = "https://bugzilla.redhat.com/show_bug.cgi?id=1300177"
+                    variants:
+                        - image_name_only:
+                            dest_path = "image" 
+                        - parent_dir:
+                            dest_path = "../image"
+                        - current_dir:
+                            dest_path = "./image"
```
Comment (will-Do) [inline libvirt/tests/cfg/virsh_cmd/domain/virsh_blockcopy.cfg]: Just to confirm that we don't have to prepare these dirs before testing, right?

Comment (qiankehan) [inline libvirt/tests/cfg/virsh_cmd/domain/virsh_blockcopy.cfg]: Yes. These cases are about relative dir, so no need to prepare the dirs. But there are another concern: if the images are created, in other words, test failed, the scrips need to remove the images created.
I will add these codes later.

Comment (qiankehan) [inline libvirt/tests/cfg/virsh_cmd/domain/virsh_blockcopy.cfg]: I found the code of cleaning the images created. No need to add codes

**Comment by balamuruhans** [comment]
@will-Do Thanks

**Pull request merged**